connect/runtime/src/test/java/org/apache/kafka/connect/storage/FileOffsetBackingStoreTest.java

```diff
@@ -123,12 +127,23 @@ public void testSaveRestore() throws Exception {
 
     @Test
     public void testThreadName() {
+        converter = mock(Converter.class);
+        store = new FileOffsetBackingStore(converter);
+        tempFile = assertDoesNotThrow(() -> File.createTempFile("fileoffsetbackingstore", null));
+        Map<String, String> props = new HashMap<>();
+        props.put(StandaloneConfig.OFFSET_STORAGE_FILE_FILENAME_CONFIG, tempFile.getAbsolutePath());
+        props.put(StandaloneConfig.KEY_CONVERTER_CLASS_CONFIG, "org.apache.kafka.connect.json.JsonConverter");
+        props.put(StandaloneConfig.VALUE_CONVERTER_CLASS_CONFIG, "org.apache.kafka.connect.json.JsonConverter");
+        config = new StandaloneConfig(props);
+        store.configure(config);
+        store.start();
         assertTrue(((ThreadPoolExecutor) store.executor).getThreadFactory()
```

Comment:
Maybe we can move this `assert` to `setup` and then remove this test case. That is more simple to me